pkg/kinflate/util/util.go

```diff
@@ -111,6 +112,53 @@ func Encode(in map[GroupVersionKindName]*unstructured.Unstructured) ([]byte, err
 	return buf.Bytes(), nil
 }
 
+// SelectByGVK returns true if selector is a superset of in; otherwise, false.
+func SelectByGVK(in schema.GroupVersionKind, selector *schema.GroupVersionKind) bool {
+	if selector == nil {
+		return true
+	}
+	if len(selector.Group) > 0 {
+		if in.Group != selector.Group {
+			return false
+		}
+	}
+	if len(selector.Version) > 0 {
+		if in.Version != selector.Version {
+			return false
+		}
+	}
+	if len(selector.Kind) > 0 {
+		if in.Kind != selector.Kind {
+			return false
+		}
+	}
+	return true
+}
+
+func CompareMap(m1, m2 map[GroupVersionKindName]*unstructured.Unstructured) error {
```

Comment:
If CompareMap is a test-only utility it should be in a _test.go file and should be private.

If it is meant to be used outside tests, then it needs its own individual test and godoc.